src/ja/user/config/add.py

```diff
@@ -30,7 +30,7 @@ def blocking(self) -> bool:
         return self._blocking
 
     def __dir__(self) -> Iterable[str]:
-        return ["_blocking", "_attach", "_job"]
+        return ["_blocking", "_attach", "_job", "_ssh_config"]
```

Comment:
Wasn't there also verbosity?

Reply:
I was thinking it will be the same for all ad commands but you are right.